packages/sentry/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@debugr/sentry",
+  "version": "3.0.0-rc.1",
+  "description": "Sentry log handler for Debugr",
+  "author": {
+    "name": "Dan Kadera",
+    "email": "dan.kadera@cdn77.com"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=12.0"
+  },
+  "dependencies": {
+    "@debugr/core": "^3.0.0-rc.10",
+    "@sentry/node": "^7.33.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/uuid": "^8.3.4"
+  },
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
+  "build": {
+    "src": "src/index.ts",
+    "externals": [
+      "@elastic/elasticsearch",
+      "async_hooks",
+      "uuid"
+    ]
```

Comment:
tady by místo `@elastic/elasticsearch` mělo bejt `@sentry/node` a `async_hooks` by tam neměly bejt vůbec když je ta package nepoužívá

Reply:
a vlastně nepoužívá ani `uuid`, takže to může taky zmizet, včetně `dependencies` a `devDependencies`

Reply:
fair, fixed